src/ai/backend/manager/api/vfolder.py

```diff
@@ -1824,6 +1828,7 @@ async def clone(request: web.Request, params: Any, row: VFolderRow) -> web.Respo
             'user': user_uuid,
             'group': group_uuid,
             'cloneable': params['cloneable'],
+            'bgtask_id': task_id.hex,
```

Comment:
`bgtask_id` is a UUID field.  Though this still works because `tx.UUID()` also accepts the hexademical form, I think it is better to keep the original UUID format by replacing this with `str(task_id)`.